serialzy/serializers/tensorflow.py

```diff
@@ -62,4 +77,16 @@ def _deserialize(self, source: BinaryIO, schema_type: Type, user_type: Optional[
 
     def _types_filter(self, typ: Type):
         import tensorflow as tf  # type: ignore
-        return typ in [tf.train.Checkpoint] or (inspect.isclass(typ) and issubclass(typ, tf.Module))
+        return (
+            typ in [tf.train.Checkpoint] or
+            (inspect.isclass(typ) and issubclass(typ, tf.Module)) or
+            self._is_generic_user_object(typ)
+        )
+
+    @staticmethod
+    def _is_generic_user_object(typ: Type) -> bool:
+        return (
+            inspect.isclass(typ) and
+            getattr(typ, "__module__") == "tensorflow.python.saved_model.load" and
```

Comment:
eh, it makes sense to change dot to getattr if you passing third "default" argument to getattr